charts/px-backup/templates/px-central-oidc/pxcentral-keycloak.yaml

```diff
@@ -267,12 +267,9 @@ spec:
           image: {{ printf "%s/%s/%s:%s" .Values.images.keycloakBackendImage.registry .Values.images.keycloakBackendImage.repo .Values.images.keycloakBackendImage.imageName .Values.images.keycloakBackendImage.tag }}
           imagePullPolicy: {{ .Values.images.pullPolicy }}
           {{ if eq $operatorToChartUpgrade true }}
-          {{- if $isOpenshiftCluster}}
```

Comment:
what about fresh install on an openshift cluster. That will fail, as you wont add the security context?

Reply:
For fresh install using helm chart we don't need podSecurity context, if `operatorToChartUpgrade` is set to `true` then only we need this.